### PR TITLE
Assignment_A02

### PR DESCRIPTION
**Pull request by Priyanka15-12**

### CODE REVIEW

**Comment by Priyanka15-12** [comment]
@MetaNaveen  Pushed code.

File: Program.cs [hunk 1]
```diff
@@ -1,7 +1,33 @@
...
-        }
-    }
-}
\ No newline at end of file
+﻿internal class Program {
+   private static void Main (string[] args) {
+      int a = 1;
+      int z = 101;
+      Random r = new ();
+      int n = r.Next (a, z);
```
Comment (MetaNaveen) [inline Program.cs:6]: You could make z = 100 and use z + 1 since you are referencing z in other output statements.

File: Program.cs [hunk 2]
```diff
@@ -1,7 +1,33 @@
...
+      int attempt = 0;
+      int p = 0;
+      while (p != n) {
+         // get user input as string
+         Console.Write ("Guess number: ");    
+         // Type cast - convert string to int
+         if (int.TryParse (Console.ReadLine (), out p)) {
+            // check if non-positive and print msg
+            if (p < a || p > z)
+               Console.WriteLine ("Enter the number within the range.");
```
Comment (MetaNaveen) [inline Program.cs:19]: you should "continue" to skip the remaining statements as those will also be executed if the number is not in the range.

File: Program.cs [hunk 3]
```diff
@@ -1,7 +1,33 @@
...
-    internal class Program {
-        static void Main (string[] args) {
-            Console.WriteLine ("Hello, World!");
-        }
-    }
-}
\ No newline at end of file
+﻿internal class Program {
+   private static void Main (string[] args) {
+      int a = 1;
```
Comment (MetaNaveen) [inline Program.cs:3]: You could combine multiple variable declarations of same data type like "int a = 1, z = 101;"

**Review by MetaNaveen — CHANGES_REQUESTED** [review]
Good. It's almost done. Just fewer changes before going into next exercise.

**Comment by MetaNaveen** [comment]
Reopen.

**Comment by Priyanka15-12** [comment]
.

**Pull request merged**